app/src/main/java/com/mashup/mobalmobal/ui/profile/presenter/ProfileViewModel.kt

```diff
@@ -0,0 +1,50 @@
+package com.mashup.mobalmobal.ui.profile.presenter
+
+import com.funin.base.funinbase.base.BaseViewModel
+import com.funin.base.funinbase.extension.rx.observeOnMain
+import com.funin.base.funinbase.extension.rx.subscribeWithErrorLogger
+import com.funin.base.funinbase.rx.schedulers.BaseSchedulerProvider
+import com.mashup.mobalmobal.ui.profile.data.dto.toProfileItems
+import com.mashup.mobalmobal.ui.profile.data.repository.ProfileRepository
+import com.mashup.mobalmobal.ui.profile.domain.model.ProfileItem
+import dagger.hilt.android.lifecycle.HiltViewModel
+import io.reactivex.subjects.BehaviorSubject
+import io.reactivex.subjects.PublishSubject
+import java.io.IOException
+import javax.inject.Inject
+
+@HiltViewModel
+class ProfileViewModel @Inject constructor(
+    private val schedulerProvider: BaseSchedulerProvider,
```

Comment:
schedulerProvider를 단순히 BaseViewModel의 생성자에만 사용하시는거면 private val 없애주세요!
```suggestion
    schedulerProvider: BaseSchedulerProvider,
```

Reply:
수정했습니다~
커밋넘버 : 5dd4289